model-optimizer/mo/utils/versions_checker.py

```diff
@@ -82,6 +82,20 @@ def parse_and_filter_versions_list(required_fw_versions, version_list, env_setup
                 # this python_version requirement is not satisfied to required environment
                 # and requirement for a dependency will be skipped
                 return version_list
+        elif env_marker == 'sys_platform':
+            splited_env_req[1] = splited_env_req[1].strip(' ').replace('\'', '')
+            if '==' in env_req:
+                if not sys.platform == splited_env_req[1]:
+                    # this sys_platform requirement is not satisfied to required environment
+                    # and requirement for a dependency will be skipped
+                    return version_list
+            elif '!=' in env_req:
+                if not sys.platform != splited_env_req[1]:
+                    # this sys_platform requirement is not satisfied to required environment
+                    # and requirement for a dependency will be skipped
+                    return version_list
+            else:
+                log.error("Error during platform version check")
```

Comment:
I propose to record `sys_platform` to `env_setup` during `env-setup` initialization and use it.

Reply:
Done, now cases with several markers handled correctly.